init_snap

```diff
@@ -16,3 +16,7 @@ chmod 755 /opt/snap/bin/snapd
 chmod 755 /opt/snap/bin/snapctl
 [ -f /usr/local/bin/snapd ] || ln -s /opt/snap/bin/snapd /usr/local/bin/snapd
 [ -f /usr/local/bin/snapctl ] || ln -s /opt/snap/bin/snapctl /usr/local/bin/snapctl
+[ -f /opt/snap/bin/snapteld ] || ln -s /opt/snap/bin/snapd /opt/snap/bin/snapteld
```

Comment:
I was going to defer the change when we get the binaries updated:

from:
```
ADD ${SNAP_URL}/snap/${SNAP_VERSION}/linux/x86_64/snapd  /opt/snap/bin/snapd
ADD ${SNAP_URL}/snap/${SNAP_VERSION}/linux/x86_64/snapctl  /opt/snap/bin/snapctl
```

to:
```
ADD ${SNAP_URL}/snap/${SNAP_VERSION}/linux/x86_64/snapteld /opt/snap/bin/snapteld
ADD ${SNAP_URL}/snap/${SNAP_VERSION}/linux/x86_64/snaptel  /opt/snap/bin/snaptel
```

This PR is to unblock SDI-2201 and I intend to revisit this when SDI-2199 is complete.